kerckhoff/packages/google_drive_actions.py

```diff
@@ -79,17 +79,9 @@ def check_span(tag, name, value):
                 data = session.get(GOOGLE_API_PREFIX + "/v2/files/" + aml['id'] + "/export", params={"mimeType": "text/plain"})
                 aml_text = data.content.decode('utf-8')
         #print("IN ARCHIEML ")
-        aml_content = archieml.loads(aml_text)
-        
-        # HACK: Fixes a bad bug in the archieml parser
-        for key, value in aml_content.items():
-            if isinstance(value, list):
-                for index, item in enumerate(value):
-                    if isinstance(item, dict):
-                        if item.get("type") is None and item.get("value") and isinstance(value[index-1], dict) and value[index-1].get("type"):
-                            aml_content[key][index-1]["value"] = item["value"]
-                            aml_content[key][index] = None
-                aml_content[key] = [ i for i in aml_content[key] if i is not None ]
+
+        parser = Parser()
+        aml_content = parser.parse(aml_text)
```

Comment:
I noticed that the parser has several assert statements, that when triggered, will cause the server to return a 500 response without any clear indication of what went wrong. Could you wrap this in a try/except block and return a more informative response to the client? (If possible, it would be best if you could replace each "assert" statement in the parser with an exception with a custom error message, but if you don't completely understand the parser code then it's fine not to have this).

We will have to add the ability to show the error messages for non-200 responses in a future issue.